lib/srv/desktop/tdp/proto.go

```diff
@@ -1418,6 +1471,17 @@ func writeUint64(b *bytes.Buffer, v uint64) {
 	b.WriteByte(byte(v))
 }
 
+// tdpMaxNotificationMessageLength is somewhat arbitrary, as it is only sent *to*
+// the browser (Teleport never receives this message, so won't be decoding it)
+const tdpMaxNotificationMessageLength = 10240
+
+// tdpMaxPathLength is currently just an arbitrary value, more research is required
```

Comment:
An arbitrary value for _what_ though?

(I assume this is the lenght of a file path, but I would say that in the comment)